### DISCUSSION TITLE
Release 1.0.2

### DISCUSSION THREAD

**Discussion by lyskouski** [lyskouski/app-finance#124] [Announcements]
## [BF] Bug Fix(es)
- [#117] [RF] Revise Flow Materials
  - Concurrent saving of Currencies
  - Currency recalculations (not working anymore)
  - Budget Category shift (recalculation is not happened)

<hr /><em>This discussion was created from the release <a href='https://github.com/lyskouski/app-finance/releases/tag/v1.0.2'>Release 1.0.2</a>.</em>

**Comment by msalman360** [comment]
Hi sir,
Can the name of the Application be changed? If possible kindly guide me. Thanks

**Reply by lyskouski** [reply]
Hello,
It's been changed already to "Fingrom" -- check the latest release: https://github.com/lyskouski/app-finance/releases/tag/v1.1.0
Kind regards

**Comment by msalman360** [comment]
okay thanks

**Comment by msalman360** [comment]
Hi,
I am facing this type or issues:
e: /Users/apple/Desktop/development/app-finance-main/build/package_info_plus/.transforms/1335390f56eda7e6a23baf0188ca581a/transformed/out/jars/classes.jar!/META-INF/package_info_plus_release.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /Users/apple/.gradle/caches/modules-2/files-2.1/androidx.annotation/annotation-jvm/1.6.0/a7257339a052df0f91433cf9651231bbb802b502/annotation-jvm-1.6.0.jar!/META-INF/annotation.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /Users/apple/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk7.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /Users/apple/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /Users/apple/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib/1.8.22/636bf8b320e7627482771bbac9ed7246773c02bd/kotlin-stdlib-1.8.22.jar!/META-INF/kotlin-stdlib-jdk8.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
e: /Users/apple/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlin/kotlin-stdlib-common/1.8.22/1a8e3601703ae14bb58757ea6b2d8e8e5935a586/kotlin-stdlib-common-1.8.22.jar!/META-INF/kotlin-stdlib-common.kotlin_module: Module was compiled with an incompatible version of Kotlin. The binary version of its metadata is 1.8.0, expected version is 1.6.0.
Kindly guide me. thanks

**Reply by lyskouski** [reply]
Has it been taken from a released package or compiled by own from resources?

**Reply by lyskouski** [reply]
In case of resources usages, upgrade your environment to Kotlin 1.8.10 / Gradle 8.0
